test/functional/neutronless/l2adjacent/test_disconnected_service_creation.py

```diff
@@ -166,10 +175,13 @@ def setup_l2adjacent_test(request, bigip, makelogdir):
 
 def handle_init_registry(bigip, icd_configuration,
                          create_mock_rpc=create_default_mock_rpc_plugin):
+    print(type(bigip))
+    from pprint import pprint as pp
     init_registry = register_device(bigip)
+    pp(bigip.raw)
```

Comment:
Are lines 178-179. 191 for debug and can now be removed?  I don't see the value of dumping the raw object, except maybe in case of an error.